bigquery/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud"
+
+describe "BigQuery Quickstart" do
+
+  it "creates a new dataset" do
+    # Initialize test objects
+    gcloud_test_client = Google::Cloud.new ENV["GOOGLE_CLOUD_PROJECT"]
```

Comment:
Small #nit  -  Align variable assignments

Alignment of variable assignments is a common Ruby idiom and it is *particularly* useful for improving the readability of our code samples.  So let's try to be consistent and always align (when it makes sense to do so)